cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -35,10 +35,16 @@ type MasterConfiguration struct {
 	KubeletConfiguration KubeletConfiguration
 	Networking           Networking
 	KubernetesVersion    string
-	CloudProvider        string
 	NodeName             string
 	AuthorizationModes   []string
 
+	// Name of the cloud provider
+	CloudProvider string
```

Comment:
The CloudProvider strings don't really need to be shuffled around, it makes it look like you're changing it for some reason.

Reply:
Fixed. thanks!